specification/desktopvirtualization/resource-manager/readme.ruby.md

```diff
@@ -13,6 +13,28 @@ azure-arm: true
 ``` yaml $(ruby) && $(multiapi)
 batch:
   - tag: package-2019-01-23-preview
+  - tag: package-2019-09-24-preview
+  - tag: package-2019-12-10-preview
+```
+
+### Tag: package-2019-12-10-preview and ruby
+
+These settings apply only when `--tag=package-2019-12-10-preview --ruby` is specified on the command line.
+Please also specify `--ruby-sdks-folder=<path to the root directory of your azure-sdk-for-ruby clone>`.
+
+``` yaml $(tag) == 'package-2019-12-10-preview' && $(ruby)
+namespace: "Azure::DesktopVirtualization::Mgmt::V2019_12_10_preview"
+output-folder: $(ruby-sdks-folder)/management/azure_mgmt_DesktopVirtualization/lib

```

Comment:
is it ok to name the folder and the package name to `azure_mgmt_desktop_virtualization`?

Reply:
also in line 6

Reply:
ok